packages/cli/src/commands/build.ts

```diff
@@ -53,6 +53,8 @@ export default class Build extends Command {
         }
       }
 
+      //Checking need to be done before cli.action,otherwise error logs won't display
+      await typescriptBuildCheck(directory);
       cli.action.start('Building and packing code');
```

Comment:
I think this suppresses any other logging, we should be able to get webpack to output TS errors but we might need to disable this line.